Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '?,?,?,?,?,NOW(),NOW())' at line 1
    at Sequence._packetToError (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/sequences/Sequence.js:47:14)
    at Query.ErrorPacket (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/sequences/Query.js:79:18)
    at Protocol._parsePacket (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/Parser.js:43:10)
    at Protocol.write (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/Protocol.js:38:16)
    at Socket.<anonymous> (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/Connection.js:88:28)
    at Socket.<anonymous> (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/Connection.js:526:10)
    at Socket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:559:12)
    --------------------
    at Protocol._enqueue (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/Protocol.js:144:48)
    at Connection.query (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/Connection.js:198:25)
    at Object.query (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/helpers/db_helper.js:68:16)
    at /Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/controllers/login_controller.js:53:24
    at Query.<anonymous> (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/helpers/db_helper.js:69:24)
    at Query.<anonymous> (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/Connection.js:526:10)
    at Query._callback (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/Connection.js:488:16)
    at Sequence.end (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
    at Query._handleFinalResultPacket (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/sequences/Query.js:149:8)
    at Query.EofPacket (/Users/subhamkumarsahoo/Desktop/NODE/OnlineGrocery/node_modules/mysql/lib/protocol/sequences/Query.js:133:8)


    this crash log came due to i did not make test user in db but tried to access that user in order to see if we use same username it says the username is already exists 
    